### PR TITLE
Instant-forking CPM version for testing

### PR DESCRIPTION
**Pull request by edmundnoble**
This is a new `Chainweb.Version` which enables all forks immediately, similar to fast-devnet, but for testing. Lots of tests would prefer using this over `fastForkingCpmTestVersion`. However, doing this has revealed an odd race in `RemotePactTest`. I include a failed attempt at a mitigation here, which is to wait until warp has begun serving the service and P2P APIs.

### CODE REVIEW

File: src/Chainweb/Chainweb.hs [hunk 1]
```diff
@@ -634,15 +635,23 @@ mkThrottler e rate c = initThrottler (defaultThrottleSettings $ TimeSpec (ceilin
 -- -------------------------------------------------------------------------- --
 -- Run Chainweb
 
+data NowServing = NowServing
```
Comment (chessai) [inline src/Chainweb/Chainweb.hs:638]: :nail_care:

File: src/Chainweb/Version.hs [hunk 2]
```diff
@@ -305,6 +305,9 @@ data Upgrade = Upgrade
     deriving stock (Generic, Eq)
     deriving anyclass (NFData)
 
+instance Show Upgrade where
+    show _ = "<upgrade>"
```
Comment (chessai) [inline src/Chainweb/Version.hs:309]: why can (or should) this not derive Show?

Comment (edmundnoble) [reply src/Chainweb/Version.hs:309]: Upgrade transactions are *massive blobs* of base64-encoded text and are probably not useful. Maybe we should actually label them with names instead to get a decent show...

**Comment by edmundnoble** [comment]
Closed in favor of #1866

**Pull request closed without merging**